bitnami/discourse/templates/service.yaml

```diff
@@ -34,6 +34,6 @@ spec:
       nodePort: null
       {{- end }}
     {{- if .Values.service.extraPorts }}
-    {{- include "common.tplvalues.render" (dict "value" .Values.service.extraPorts "context" $) | indent 4 }}
+    {{- toYaml .Values.ingress.extraPorts | nindent 4 }}
```

Comment:
`.Values.ingress.extraPorts` does not exist in the values.yaml as a parameter

Reply:
Forgot the switch "ingress" with "service", but the issue still stands. 

I cannot get the service.extraPorts field to template properly using the bitnami.common.tplvalues function.  

I found   "{{- toYaml .Values.ingress.extraPorts | nindent 4 }}" in the same "bitnami/discourse" chart, in the ingress.yml file for a similar field (ingress.extraTLS).  This field actually works though, so I believe switching this change would fix it.

Reply:
Check my suggestion below, I checked it and seems to work just changing indent by nindent